### PR TITLE
Backports 0.19 pr8

### PR DESCRIPTION
**Pull request by Munkybooty**
All was well until feature_llmq_data_recovery.py started to fail. 
I'm unsure of the cause but i'm assuming it had something to do with #16578.

### CODE REVIEW

**Comment by Munkybooty** [comment]
Oh that's very odd, I guess it only failed on my machine. Disregard the previous comment then

File: src/interfaces/chain.h [hunk 1]
```diff
@@ -15,13 +15,15 @@
...
 class CBlock;
+class CFeeRate;
 class CRPCCommand;
 class CScheduler;
 class CValidationState;
 class CFeeRate;
 class CBlockIndex;
 class Coin;
 class uint256;
+enum class RBFTransactionState;
```
Comment (UdjinM6) [inline src/interfaces/chain.h:26]: 15531: we have no rbf
```suggestion
```

File: test/functional/rpc_psbt.py [hunk 2]
```diff
@@ -17,7 +17,10 @@ def set_test_params(self):
         self.setup_clean_chain = False
         self.num_nodes = 3
        # TODO: remove -txindex. Currently required for getrawtransaction call.
-        self.extra_args = [[], ["-txindex"], ["-txindex"]]
+        self.extra_args = [[],
+            ["-txindex"],
+            ["-txindex"]
```
Comment (UdjinM6) [inline test/functional/rpc_psbt.py:22]: 15911: missing multiple updates for `conf_target` in `rpcwallet.cpp`

File: src/interfaces/node.cpp [hunk 3]
```diff
@@ -335,6 +335,7 @@ class NodeImpl : public Node
         return GuessVerificationProgress(Params().TxData(), tip);
     }
     bool isInitialBlockDownload() override { return ::ChainstateActive().IsInitialBlockDownload(); }
+    bool isAddressTypeSet() override { return !::gArgs.GetArg("-addresstype", "").empty(); }
```
Comment (UdjinM6) [inline src/interfaces/node.cpp:338]: 16497: looks useless for us (we have no `-addresstype` and no bech32 addresses). maybe DNM?

File: src/qt/dash.cpp [hunk 4]
```diff
@@ -195,8 +195,11 @@ void BitcoinCore::shutdown()
     }
 }
 
-BitcoinApplication::BitcoinApplication(interfaces::Node& node, int &argc, char **argv):
-    QApplication(argc, argv),
+static int qt_argc = 1;
+static const char* qt_argv = "bitcoin-qt";
```
Comment (UdjinM6) [inline src/qt/dash.cpp:199]: 16578: dashify

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/interfaces/node.cpp (hunk 3 above)
Comment (Munkybooty) [reply src/interfaces/node.cpp]: if that is the case should i also DNM #15711 on our sheet as it is the first set of changes? looks nothing more than a syntax change to call bech32 addresses `legacy addresses`

Comment (UdjinM6) [reply src/interfaces/node.cpp]: yes, I guess so

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**